lib/src/map/map.dart

```diff
@@ -134,7 +134,7 @@ class MapState {
     if (!bounds.isValid) {
       throw Exception('Bounds are not valid.');
     }
-    var target = _getBoundsCenterZoom(bounds, options);
+    var target = getBoundsCenterZoom(bounds, options);
```

Comment:
I think a better way would be to expose this method on `MapControllerImpl` and `MapController` instead. Their purpose is to expose the methods on `MapState`

Reply:
Right, my bad. now its ok

Reply:
I see, thanks for the clarification.